src/portal.py

```diff
@@ -242,6 +284,10 @@ def portal_update(
         new_attr_dict["mail"] = mails
 
```

Comment:
At the very end of this function, we return a dict with fullname, mailalias, mailforward

We probably want to add the main email to be consistent

Reply:
Thanks, done